src/taipy/core/config/data_node_config.py

```diff
@@ -509,3 +522,53 @@ def _configure_sql(
         )
         Config._register(section)
         return Config.sections[DataNodeConfig.name][id]
+
+    @staticmethod
+    def _configure_mongo_collection(
+        id: str,
+        db_username: str,
+        db_password: str,
+        db_name: str,
+        read_query: str,
+        collection_name: str = None,
+        db_port: int = 27017,
+        db_host: str = "localhost",
+        exposed_type=_EXPOSED_TYPE_PANDAS,
```

Comment:
We don't want to have pandas as the default exposed type. I believe the exposed_type should be required, as a DataClass provided by the user.

```suggestion
        exposed_type: Any,
```